### PR TITLE
Support Splinter exporters (#555)

### PR DESCRIPTION
**Pull request by Allanbeddouk**
Add Splinter model in exporters.

Contributions related to @michaelbenayoun 's checklist from issue #555

### CODE REVIEW

File: tests/exporters/exporters_utils.py [hunk 1]
```diff
@@ -176,6 +176,7 @@
     "roformer": "junnyu/roformer_chinese_base",
     "segformer": "nvidia/segformer-b0-finetuned-ade-512-512",
     "squeezebert": "squeezebert/squeezebert-uncased",
+    "splinter": "hf-internal-testing/tiny-random-SplinterModel",
```
Comment (fxmarty) [inline tests/exporters/exporters_utils.py:179]: Can you add it in `PYTORCH_EXPORT_MODELS_TINY` as well? It's actually this dict that is used for testing.

**Review by fxmarty — COMMENTED** [review]
Thank you for your contribution!

**Review by michaelbenayoun — APPROVED** [review]
LGTM once @fxmarty has been addressed and the tests pass.

File: tests/exporters/exporters_utils.py (hunk 1 above)
Comment (Allanbeddouk) [reply tests/exporters/exporters_utils.py:179]: Yes you're right, my bad for not adding it there as well. Thank you for your feedback !

**Review by fxmarty — APPROVED** [review]
Thank you, the remaining two errors are unrelated, so merging!

**Pull request merged**